app/src/components/EquityOption.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react'
 import { Box, Button, GU, IconEdit, textStyle, useTheme } from '@aragon/ui'
 import { useAppState } from '@aragon/api-react'
+import { convertMultiplier } from '../utils/calculations'
```

Comment:
should we try to find a better name for this file? also what do. you think about naming all the utils files with `-utils` like we are doing in the dashboard?

Reply:
I think `calculations` is fine since it's a general and simple term. Do you have one particularly in mind? 

> also what do. you think about naming all the utils files with -utils like we are doing in the dashboard?

Yeah agree we should be consistent with the file naming format 👍